client/loading.html

```diff
@@ -15,39 +15,35 @@
       })
     </script>
     <style>
-      li {
-        color: black;
-      }
-      .pending {
-        color: grey;
+      html, body {
+        margin: 0;
+        padding: 0;
+        width: 100%;
+        height: 100vh;
+        background: linear-gradient(225.77deg, #00D1FF 23.96%, #AEFFEC 100%), #FFF;
+        display: grid;
+        place-items: center;
+        position: relative;
       }
       .loader {
-        border: 5px solid grey;
-        border-bottom-color: transparent;
+        border: 4px solid #f3f3f3;
+        border-top: 4px solid #3498db;
         border-radius: 50%;
-        display: inline-block;
-        box-sizing: border-box;
-        animation: rotation 1s linear infinite;
-      }
-      @keyframes rotation {
-      0% {
-        transform: rotate(0deg);
+        width: 20px;
+        height: 20px;
+        animation: spin 2s linear infinite;
```

Comment:
great job adding all of this in css @pdelboca

Reply:
CSS is super powerful nowadays!